src/main/java/seedu/address/model/person/Telegram.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Telegram {
+    public static final String MESSAGE_CONSTRAINTS = "Telegram can take any values, and it should not be blank";
```

Comment:
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Telegram should start with the '@' symbol, should not contain whitespace, and should not be blank";
```

We can update this message to reflect the updated regex